src/test/java/com/datapipe/jenkins/vault/it/folder/FolderIT.java

```diff
@@ -46,11 +36,19 @@
 import java.util.List;
 import java.util.Map;
 
-import hudson.model.FreeStyleBuild;
-import hudson.model.FreeStyleProject;
-import hudson.model.Result;
-import hudson.tasks.Shell;
-import jenkins.model.GlobalConfiguration;
+import static com.datapipe.jenkins.vault.it.VaultConfigurationIT.GLOBAL_CREDENTIALS_ID_1;
```

Comment:
Please don't mess with import order.

I will have a PR up shortly that fixes import/code style.